dpl_htaccess

```diff
@@ -0,0 +1,12 @@
+# dlang.org .htaccess file (local Apache configuration)
+
+# Enable mod_rewrite
+RewriteEngine On
+
+# Indicate that this directory maps to the root
+# directory of the domain.
+RewriteBase /
```

Comment:
I think `/library/` ought to work, plus it should also work for library-prerelease despite that the map file only has /library/.